docs.cask.co/scripts/build.sh

```diff
@@ -18,17 +18,22 @@
 # The configuration file is specific for the product (cdap, coopr, tigon).
 # The resulting version menu is a JavaScript file, placed in the product directory on the web server.
 # The 'version' file is a text file with the current version, placed in the product directory on the web server.
+# Re-writes the redirect-page.js script to include the types that were used.
 
 __site=$(cd $(cd $(dirname ${BASH_SOURCE[0]}); pwd -P); cd $(pwd -P)/.. ; pwd -P)  # traverse symlinks, etc
 __types=${@:-$(cd "${__site}/configs"; ls -1 | sed -e 's/.txt$//')} # support multiple types
 
 rm -rf ${__site}/target || ( echo "Could not remove target directory" ; exit 1)
 mkdir ${__site}/target || ( echo "Could not create target directory" ; exit 1)
-cp -R ${__site}/www/* ${__site}/target || ( echo "Could not copy www to target directory" ; exit 1)
+cp -R ${__site}/www ${__site}/target/www || ( echo "Could not copy www to target directory" ; exit 1)
 
+__types_array=""
 for __type in ${__types}; do
+  __types_array+=", \"${__type}\""
```

Comment:
This isn't an array. It's confusing. Also, you're doing this in the loop, where it doesn't make sense. You already have the complete list, with `${types}`, so just use that...